tests/it/homeassistant_HassTurnOff.yaml

```diff
@@ -1,34 +1,34 @@
-language: it
+language: "it"
 tests:
   - sentences:
-      - spegni l'interruttore cucina
-      - spegni interruttore cucina
-      - puoi spegnere l'interruttore cucina
-      - potresti disattivare l'interruttore cucina per favore
-      - per favore spegni interruttore cucina
-      - disattiva interruttore cucina grazie
+      - "spegni l'interruttore cucina"
+      - "spegni interruttore cucina"
+      - "puoi spegnere l'interruttore cucina"
+      - "potresti disattivare l'interruttore cucina per favore"
+      - "per favore spegni interruttore cucina"
+      - "disattiva interruttore cucina grazie"
     intent:
-      name: HassTurnOff
+      name: "HassTurnOff"
       slots:
         name: switch.cucina
```

Comment:
Here you have inconsitency. Once you used `"` and then without `"`
Should be omited